aws/resource_aws_mq_broker.go

```diff
@@ -328,6 +361,16 @@ func resourceAwsMqBrokerUpdate(d *schema.ResourceData, meta interface{}) error {
 		}
 	}
 
+	if d.HasChange("logs") {
+		_, err := conn.UpdateBroker(&mq.UpdateBrokerRequest{
```

Comment:
If there are multiple updates that call the same update API call, we generally combine the requests 👍